websites/H/HBO Max/dist/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://schemas.premid.app/metadata/1.0",
   "author": {
-    "name": "PreMiD",
-    "id": "503557087041683458"
+    "name": "Kedi",
+    "id": "238668511222693888"
```

Comment:
If you're now the author, you should of changed the client ID too

Reply:
Created new clientId 879535934977245244

Proof that it works with the new clientId:

![image](https://user-images.githubusercontent.com/4789757/130541463-7b5da564-8e99-462f-ae57-1d5509a7aacc.png)